build.yaml

```diff
@@ -714,6 +714,8 @@ filegroups:
   - src/cpp/util/time.cc
   uses:
   - grpc++_codegen_base
+  - grpc_base
+  - grpc_secure
```

Comment:
Should we use `deps: - grpc ` here? We might not need to add the source files of c-core to `libgrpc++` and `libgrpc++_unsecure`.